tests/structuredheadings/formatcombo.js

```diff
@@ -0,0 +1,76 @@
+/* bender-tags: editor,unit */
+/* bender-ckeditor-plugins: wysiwygarea,structuredheadings,toolbar,basicstyles,dialog,richcombo */
+
+// Clean up all instances been created on the page.
+(function () {
+
+  bender.editor = {
+    allowedForTests: "h1; h2; p"
+  };
+
+  bender.test({
+    setUp: function () {
+      //Anything to be run before each test if needed
+    },
+
+    "if no headings in document, the new heading is autonumbered": function () {
+      var bot = this.editorBot;
+      var ed = this.editor;
+      bot.setHtmlWithSelection("<p>^foo</p>");
```

Comment:
These tests are to test choosing "Heading 2" "Heading 3" etc. 

There were some things broken previously, where new headings would not get the related autonumbering css classes.